pom.xml

```diff
@@ -945,6 +945,7 @@
             <configuration>
               <additionalparam>-Xdoclint:none</additionalparam>
               <failOnError>false</failOnError>
+              <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
duplicate with line 946?

Reply:
good catch, guess it was done on both branches